docs/content/en/docs/tasks/cluster/cluster-core-components-upgrade.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Upgrade core components"
+linkTitle: "Upgrade core components"
+weight: 25
+date: 2017-01-05
+description: >
+  How to upgrade core components in your cluster.
+---
+EKS Anywhere `upgrade` also supports upgrading the following core components:
+
+* Core CAPI
+* CAPI providers
+* Cert-manager
+* Etcdadm CAPI provider
+* EKS Anywhere controllers and CRDs
+* GitOps controller (optional)
+
+The latest versions of these core EKS Anywhere components are embedded into a bundles manifest that the CLI uses to fetch the latest versions and image builds needed for each component upgrade.
+
+---
+### Performing a core components upgrade
+To upgrade to the latest version of the core EKS Anywhere components, run the same upgrade command as you would when upgrading the cluster:
+```
+eksctl anywhere upgrade cluster -f ./cluster.yaml
```

Comment:
This command is the same as what's in the upgrade cluster section. Is there any configuration we want to add or additional steps someone should take?

If not would it be worth merging this into the upgrade cluster section?

Reply:
No, it requires no additional configuration since we dynamically pull the latest versions from our bundles manifest. I think it's worth it to only mention this in the upgrade cluster section